models/articlesModels.js

```diff
@@ -20,7 +21,10 @@ exports.fetchArticleById = ({ article_id }) => {
       if (result.rows.length) {
         return result.rows[0];
       }
-      return Promise.reject({ status: 404, msg: "article not found" });
+      return Promise.reject({
+        status: 404,
+        msg: "article not found",
+      });
     })
     .catch((err) => {
```

Comment:
we would want to catch all our errors in either the controller or models not both